docs/source/_templates/sphinx_conf.template.py

```diff
@@ -6,6 +6,7 @@
 import sys
 import os
 import geoips
+from geoips.pydantic import sectors
```

Comment:
If this line is removed, it may cause the tests to pass.

Reply:
Thanks, @jsolbrig . It did clear one additional test.